### PR TITLE
add new company - Coodesh

### PR DESCRIPTION
**Pull request by FelipeFama**
Signed-off-by: felipe fama <91050670+FelipeFama@users.noreply.github.com>

This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [] You are an employee of the company mentioned and confirm all included details are correct.
- [] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ Name | Website | Region
 [Codea IT](/company-profiles/codea-it.md) | https://www.codeait.com | Worldwide
 [CodePen](/company-profiles/codepen.md) | https://codepen.io | Worldwide
 [CodeSandbox](/company-profiles/codesandbox.md) | https://codesandbox.io | Worldwide
+[Coodesh](/company-profiles/coodesh.md) | https://coodesh.com/ | Brazil
```
Comment (KieranRobson) [inline README.md:138]: This should be above `[Core-Apps](/company-profiles/core-apps.md) | https://www.core-apps.com/ | USA`

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Please also add a company profile, you have linked to one however it doesn't exist

Could you also fill the pull request template please?

File: README.md [hunk 2]
```diff
@@ -236,6 +237,7 @@ Name | Website | Region
 [Fire Engine Red](/company-profiles/fire-engine-red.md) | https://fire-engine-red.com/ | Worldwide
 [Fireball Labs](/company-profiles/fireball-labs.md) | https://www.fireballlabs.com | Germany
 [FivexL](/company-profiles/fivexl.md) | https://fivexl.io | Worldwide
+[Fiverr](/company-profiles/fiverr.md) | https://www.fiverr.com/ | North America, Asia
```
Comment (KieranRobson) [inline README.md:240]: Please make another pull request for `fiverr`

Comment (FelipeFama) [reply README.md:240]: okay

**Comment by FelipeFama** [comment]
> Adicione também um perfil de empresa, você vinculou a um, mas ele não existe
> 
> Você também poderia preencher o modelo de pull request, por favor?

**Comment by FelipeFama** [comment]
> Please also add a company profile, you have linked to one however it doesn't exist
> 
> Could you also fill the pull request template please?

> As well ? Does the profile have to be the part of the website about the company ?
